### PR TITLE
R doc 1570 cloud supp

### PR DESCRIPTION
**Pull request by reebhub**
fixes, improvements & structure changes

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown [hunk 1]
```diff
@@ -1,112 +1,133 @@
...
 
-{NOTE: }
-If your free instance is using more resources than the provided credit, you'll need
-to pay for these resources or stop your instance. 
-{NOTE/}
+* A free instance is identical to a development [Dev10](../cloud/cloud-instances#a-development-cloud-server) instance.  
+  It comes with a monthly credit of **10$**, used to cover the cost of the instance and of incidentals like backup storage 
+  and traffic.  
+  {NOTE: }
+  If your free instance is using more resources than the provided credit, you'll need
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown]: Maybe we should add a link to some more thourgh explanation of the credits, how are they calculated and renewed etc.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 2]
```diff
@@ -1,123 +1,258 @@
-## RavenDB on the Cloud: Overview
+# RavenDB on the Cloud: Overview
 
 {NOTE: }
 
-Running RavenDB as a cloud service takes much of its administration from your hands and passes it to 
-RavenDB's developers, allowing them to maintain and optimize your instances for you, take care of cluster 
-operations like adding or removing nodes, and run daily tasks like automatic data backup.  
+Running RavenDB as a cloud service takes much of its administration from your hands and passes it to those of
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: Maybe we need something more subtle then "takes much of itss administration from your hands"

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 3]
```diff
@@ -1,117 +1,176 @@
...
+We charge on-demand **by the minute**, with a **minimum limit of one hour**.  
+If you run an instance for 93 minutes, you will pay for exactly 93 minutes.  
+If you run it for less than an hour, say 42 minutes, you will still be charged for a full hour worth to match the minimum limit.  
+{NOTE/}
+
+---
+
+####Yearly Payment  
+
+We support two models of yearly payments: **Yearly with no upfront payment**, and **Yearly with an upfront payment**.
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]:
```suggestion
We provide two models of yearly payments: **Yearly with no upfront payment**, and **Yearly with an upfront payment**.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 4]
```diff
@@ -1,117 +1,176 @@
-## RavenDB on the Cloud: Pricing, Payment and Billing
+# RavenDB on the Cloud: Pricing, Payment and Billing
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: It feels to me like this page does not belong to the documentation, it could be found also in ravendb.net or in the portal, as an explanation page. @ppekrol @ayende ?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown [hunk 5]
```diff
@@ -1,55 +1,83 @@
...
-to access your cluster. In the cloud [Control Panel](../cloud/cloud-control-panel), open the `Products` tab and click the 
-`Download Certificate` button.  
+**1.** Go to the [Products tab](../cloud/portal/cloud-portal-products-tab) in the cloud portal and click on the `Download 
+Certificate` button.  
 
-![](images\CloudSecurity_1.png)  
+!["Download Certificate"](images\security-001-download-certificate.png "Download Certificate")  
   
-Extract the certificate package and open it. On Windows double click the `.pfx` file and the `certificate import wizard` will launch: 
+**2.** Extract the certificate package and open it. Double-click the `.pfx` file and the certificate import wizard will launch:
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown]: What about windows/linux differences. We should either give detailed explanation here, or just skip it.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown [hunk 6]
```diff
@@ -1,55 +1,83 @@
...
-  When you select the appropriate client certificate, the RavenDB management studio will launch.  
-
-This initial client certificate grants you [operator](../server/security/authorization/security-clearance-and-permissions#operator) 
-level clearance to the server.  
-To generate additional operator or [user](../server/security/authorization/security-clearance-and-permissions#user) 
-certificates, go to the server management studio.  
-Click on `Manage Certificates`:  
-
-!["Manage Certificates"](images\CloudScaling_ManageCertificates.png "Manage Certificates")  
+**3.** Click `Next` repeatedly until the wizard completes.
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown]: Maybe we should also mention that in some cases we need a certificate with a passwork protection

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+   **C**. Choose the issue's severity: Minor, Normal, or Critical.  
+   **D**. Write your question in your own words  
+   **E**. Send us your support call
+
+##Support Entitlement  
+
+Your support requests will be prioritized by their severity and your product plan.  
+
+* Support for **Free** users  
+  Our Free support includes full availability of your free cloud instance services,
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: what does full availability mean here?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-backups-tab.markdown [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+{PANEL: Backup Files }
+
+The Backups tab lists your products, their databases and their backup files.  
+!["Backup Files"](images\portal-backups-tab-002-backup-files.png "Backup Files")  
+
+You can approach each backup file and restore it.
+
+{PANEL/}
+
+##Related Articles
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-backups-tab.markdown]: We also need the backup page related article link here

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+  Use this tab's **Terminate** button to eliminate your cluster.  
+  !["Manage Product: Terminate"](images\portal-products-007-manage-terminate.png "Manage Product: Terminate")  
+
+  {NOTE: }
+  Terminating your instance is **irreversible**. Your data and cluster properties will be permanently lost.  
+  {NOTE/}
+
+{PANEL/}
+
+##Related Articles
```
Comment (maximburyak) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown]: We probably also need links to overview page and others

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 10]
```diff
@@ -1,123 +1,258 @@
...
+   - Provide your email address  
+   - Read the **terms & conditions** and the **privacy policy**, and click the **accept** checkbox if you accept them.  
+   - Click the **Next** button  
+
+---
+
+#### 2. Domain  
+  !["Registration: Domain Name"](images\registration-002-domain.png "Registration: Domain Name")  
+
+   - Enter an unoccupied domain name of your choice, that would be used for your cloud products.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: Here you need to add an example that the domain will be used for the URLS of the cluster.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 11]
```diff
@@ -1,123 +1,258 @@
...
+##Instances, Provisioning and RavenDB Products  
 
-Reserved clusters are production clusters of grade *Standard* or *Performance*.  
-Their resources are pre-allocated and can be used only by them, and they make full use of their underlying cloud resources.  
-Designed to run production systems with demanding workloads, these are the workhorses of the RavenDB Cloud.
+---
+
+#### Cloud Instances  
+
+A cloud instance is a virtual machine, allocated for us by a cloud provider like AWS or Azure.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: That isn't necessarily correct. Let's go with "a cloud instance is a ravendb instance running on a cloud provider.."

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 12]
```diff
@@ -1,123 +1,258 @@
...
+
+There are three types of RavenDB cloud products. They are explained in detail in the 
+[Tiers and Instances ](../cloud/cloud-instances) section, in the meantime - here are 
+a few words about each.  
+
+---
+
+####The Free Tier  
+This tier offers a single product: a single-node RavenDB cloud server designated for experiments and evaluation.  
+It is operated by a low-end instance that is shared with other users and is equipped with a subset of RavenDB features.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]:
```suggestion
It is operated by a low-end instance that is shared with other users and is equipped with the community license subset of RavenDB features.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 13]
```diff
@@ -1,123 +1,258 @@
...
+higher CPU load. All configurations are single-node servers, but a development server is equipped 
+with RavenDB's full features set.  
+Though superior to a free instance, a development instance still runs a single node on shared 
+hardware and is designated for development, not production.  
 
+---
+
+####The Production Tier
+A production configuration is a full RavenDB cluster of at least three nodes. You can choose its 
+components using the Portal up to 2 TB of disk space, 192 GB of RAM and 48 cores, or contact
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: I'm not sure that we want to specify those limits here. We can actually provision bigger instances, and that will change.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 14]
```diff
@@ -1,123 +1,258 @@
...
+{NOTE: }
+
+When all credits of an instance are consumed, its services are throttled.  
+For example, **Indexing may be delayed** and **Requests may be denied**.  
 
-Each RavenDB instance in a burstable cluster is given **credits**, that are consumed when it uses-up its 
-computing and I/O resources. In periods of low resource usage on the other hand, credits are accumulated 
-(up to a certain limit).  
+* One credit is the equivalent of a single minute per hour of 100% CPU-power.  
+* We provide burstable instances with a 10$-worth monthly credit as a starter.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: This is not related, and should be removed.

Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]:
```suggestion

```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 15]
```diff
@@ -1,123 +1,258 @@
...
+
+When all credits of an instance are consumed, its services are throttled.  
+For example, **Indexing may be delayed** and **Requests may be denied**.  
 
-Each RavenDB instance in a burstable cluster is given **credits**, that are consumed when it uses-up its 
-computing and I/O resources. In periods of low resource usage on the other hand, credits are accumulated 
-(up to a certain limit).  
+* One credit is the equivalent of a single minute per hour of 100% CPU-power.  
+* We provide burstable instances with a 10$-worth monthly credit as a starter.  
+* We do not charge you with money when your credits are consumed.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]:
```suggestion
* Higher tier instances start with more credits and can accrue more credits for bursts. 
```

Probably need to fix the statement

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown [hunk 16]
```diff
@@ -1,123 +1,258 @@
...
 
+* **Check your Studio warnings**  
+  Do not turn your management studio's warnings off, and do check them regularly.  
+  Your instances will warn you when their credits are about to drain up. They will also perform preemptive actions 
+  to lower resources usage.  
+* The stronger an instance is, the less prone it is to credits drainage and throttling.  
+  If your instances are repeatedly throttled, consider [upscaling](../cloud/cloud-scaling).  
+* **Cluster Credits**  
+  Basic-grade production clusters are burstable, i.e. each of their nodes is accommodated by a burstable instance.  
+  When the credits of a burstable cluster are consumed, the cluster automatically shifts the workload to nodes
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]:
```suggestion
  When the credits of a burstable node in a cluster are consumed, the cluster automatically shifts the workload to nodes 
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown (hunk 4 above)
Comment (ayende) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: I think we do need pricing discussion in the docs. That is quite important for users

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 17]
```diff
@@ -1,117 +1,176 @@
...
-  {NOTE/}
+* **Yearly with _no_ Upfront payment**  
+* **Yearly _with_ an Upfront payment**  
+
+---
+
+####On-Demand Payment  
+
+The on-demand model lets you pay for your instance **by the minute**, and come and go as you please.  
+We will measure your instance run-time **from rising to termination**, and charge you with **minute granularity**.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]:
```suggestion
We will measure your instance run-time **from provisioning to termination**, and charge you with **minute granularity**.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 18]
```diff
@@ -1,117 +1,176 @@
...
+
+We support two models of yearly payments: **Yearly with no upfront payment**, and **Yearly with an upfront payment**.  
+Both grant you discounts in relation to the on-demand model.  
+
+**Be aware** however that once a yearly contract has been activated, it is charged fully to the end of the year 
+and cannot be revoked, halted or changed.  
+
+{NOTE: }
+
+* Terminating a product you've been paying for yearly, will **not** stop your monthly payments.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]:
```suggestion
* Terminating a product with a yearly contact, will **not** stop your future payments.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 19]
```diff
@@ -1,117 +1,176 @@
...
+Both grant you discounts in relation to the on-demand model.  
+
+**Be aware** however that once a yearly contract has been activated, it is charged fully to the end of the year 
+and cannot be revoked, halted or changed.  
+
+{NOTE: }
+
+* Terminating a product you've been paying for yearly, will **not** stop your monthly payments.  
   
+* Upscaling or downscaling a product you're paying for yearly, will **not** revoke the yearly agreement.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]:
```suggestion
* Upscaling or downscaling a product with a yearly contract will **not** revoke the yearly agreement.  
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 20]
```diff
@@ -1,117 +1,176 @@
...
+####Viewing current configuration price  
+While [provisioning or editing](../cloud/portal/cloud-portal-products-tab) your product, 
+costs are shown in the **Your Order** slot.  
+
+The overall price is an outcome of the hardware components you chose and your pricing model, 
+with possible [additional expenses](../cloud/cloud-pricing-payment-billing#additional-expenses).  
 
-  ---
+See for example the differences in price between the production configurations PB0 and PB1:  
+![PB0](images/pricing_001_PB0.png "PB0")
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: PB10 , you meant?

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown [hunk 21]
```diff
@@ -1,117 +1,176 @@
...
-  Your charges for traffic to and from your instance will be added to your basic pricing plan.  
-  Find traffic fees in your 
+  Charges for traffic to and from your instance will be added to your basic pricing plan.  
+
 * **Expanding your Disk Storage**  
-  We will automatically expand the amount of disk space allocated for your instance when its usage reaches 90%, 
-  to prevent any chance of data loss. The new disk size will remain (it won't shrink back to its initial size if 
-  you delete your data), and your payment course will be updated accordingly.  
+  We will automatically expand the amount of disk space allocated for your product when its usage reaches 90%, 
+  to prevent any chance of data loss. The new disk size will remain, and your payment plan will be updated accordingly.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]:
```suggestion
  to avoid running out of disk space Your payment plan will be updated to reflect the new storage size.
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown [hunk 22]
```diff
@@ -1,55 +1,108 @@
...
+Once the certificate is imported, click your cloud instance's URL.  
+Your browser will prompt you to select a certificate. When you select the client certificate, the RavenDB [management studio]() will launch.  
+
+{PANEL/}
+
+{PANEL: Using Additional Certificates}
+
+The initial client certificate grants you [operator](../server/security/authorization/security-clearance-and-permissions#operator) 
+clearance to the server.  
+RavenDB cloud products do not give you the highest clearance level,
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown]: I don't like the terminology here.
Better to say that you are getting an operator level certificate, which allow you to perform operations such as creating / deleting databases, managing access to the cluster, the ability to inspect the state of the cluster, etc.

You can mention that you cannot add nodes to the cluster, but let's not dwell into that.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 7 above)
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]:
```suggestion
  Our Free support includes availability of your free cloud instance services, 
```

Comment (ayende) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: Important, there is support bundled with the _paid_ instances here, which focus on getting them up and running, etc.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+
+##Support Entitlement  
+
+Your support requests will be prioritized by their severity and your product plan.  
+
+* Support for **Free** users  
+  Our Free support includes full availability of your free cloud instance services, 
+  response to any connectivity issues, and your participation in mailing lists and forums.  
+* **Professional**  
+  We also support bug fixes, email and phone support, and access to RavenDB's core developers.
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: Additional monitoring of your system, focus on building better RavenDB applications, architectural guidance, etc.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown [hunk 24]
```diff
@@ -14,54 +16,73 @@
...
+{PANEL/}
 
-### Backup Encryption  
+{PANEL: Backup Encryption}
 
 Backup files are always encrypted.
 
-* If the database being backed up is NOT encrypted:  
-  We will encrypt it using an encryption key that **we** manage, unique to your account.  
+* If your database is NOT encrypted:
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown]: fixed

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown [hunk 25]
```diff
@@ -14,54 +16,73 @@
...
-### Charging For Your Backup storage  
+{PANEL: Charging For Your Backup storage}
 
-Backup storage up to 1 GB is free. Your monthly average usage will be measured, and you'll be charged accordingly.  
+Backup storage up to 1 GB is free. Your backup storage usage will be measured once a day. Each month you will be charged 
+based on the *average* amount of storage you used per day.
 
 {NOTE: } 
 Backup files are compressed.  
+To ensure that your data is safe and can always be restored if necessary, **we do not delete backup files before 14 days have passed.**
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown]: added

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown [hunk 26]
```diff
@@ -14,54 +16,73 @@
...
-### Custom Backup  
+{PANEL: Custom Backup}
 
 You can create your own [ongoing backup tasks](https://ravendb.net/docs/article-page/4.2/Csharp/studio/database/tasks/ongoing-tasks/backup-task) 
 on your RavenDB cloud instance as you would off-cloud.  
 
+However, unlike an on-premises instance of RavenDB, backup files **cannot be saved locally** due to storage limitations. Cloud 
+providers charge for sending data out of the cloud, therefore saving backup files externally is limited to the 
+[development](../cloud/cloud-instances#a-development-cloud-server) and [production](../cloud/cloud-instances#a-production-cloud-cluster) tiers.  
+If you have access to another service by the same cloud provider (such as an S3 Bucket if your cloud provider is AWS), you can save your backup
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown]: section remade

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown]: done (enhanced "overview"-->"credits")

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-overview.markdown]: done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: done (offer)

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 7 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: when you add a new product, "Availability of Cloud Service" is the first item in the "RavenDB Cloud support options" table. As I understand it, a cloud server doesn't require as much support regarding availability.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-backups-tab.markdown (hunk 8 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-backups-tab.markdown]: true. done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown (hunk 9 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-products-tab.markdown]: Added the overview article for now
Will add others if I think of any, open to ideas.. (o:

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown [hunk 27]
```diff
@@ -1,67 +1,94 @@
...
 
-### Custom Backup  
+{PANEL: Custom Backup}
 
 You can create your own [ongoing backup tasks](https://ravendb.net/docs/article-page/4.2/Csharp/studio/database/tasks/ongoing-tasks/backup-task) 
-on your RavenDB cloud instance as you would off-cloud.  
+on your RavenDB cloud instance, as you would off-cloud.  
+
+However, unlike an on-premises instance of RavenDB, backup files **cannot be saved on your instance's cloud host**, due to storage limitations. 
+Cloud providers therefore send your backup files out of the cloud, and charge you for it.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown]: done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown [hunk 28]
```diff
@@ -1,67 +1,94 @@
...
 
 You can create your own [ongoing backup tasks](https://ravendb.net/docs/article-page/4.2/Csharp/studio/database/tasks/ongoing-tasks/backup-task) 
-on your RavenDB cloud instance as you would off-cloud.  
+on your RavenDB cloud instance, as you would off-cloud.  
+
+However, unlike an on-premises instance of RavenDB, backup files **cannot be saved on your instance's cloud host**, due to storage limitations. 
+Cloud providers therefore send your backup files out of the cloud, and charge you for it.  
+
+If you have access to another service by the same cloud provider (such as an S3 Bucket if your cloud provider is AWS), you can save your backup 
+files there free of charge.
```
[the remark this answers is not in the document]
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-backup.markdown]: done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown (hunk 20 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: yup, thx

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown (hunk 21 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-pricing-payment-billing.markdown]: When the disk space allocated for your product is 90% full, we will expand it to avoid running out of storage.  
Your payment plan will be updated to reflect the new storage size.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown (hunk 22 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-security.markdown]: done

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 7 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: * Support for **Free** users  
  Our Free support includes the availability of your free cloud instance services, 
  response to any connectivity issues, and your participation in mailing lists and forums.  
* **Professional**  
  Paid products get all the support they need to get their instances up and running.  
  We also support bug fixes, email and phone support, and access to RavenDB's core developers.  
* **Production**  
  24/7 Support.

File: Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown (hunk 23 above)
Comment (reebhub) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/portal/cloud-portal-support-tab.markdown]: * Support for **Free** users  
  Our Free support includes the availability of your free cloud instance services, 
  response to any connectivity issues, and your participation in mailing lists and forums.  
* **Professional**  
  Paid products get all the support they need to get their instances up and running.  
  Additional monitoring of your system, focus on building better RavenDB applications, and architectural guidance.  
  We also provide bug fixes, email and phone support, and access to RavenDB's core developers.  
* **Production**  
  24/7 Support.

**Comment by reebhub** [comment]
Reopening a new one after rebase
